src/sidebar/components/help-panel.js

```diff
@@ -105,34 +105,30 @@ function HelpPanel({ auth, session }) {
         </h3>
         <div className="help-panel__footer">
           {activeSubPanel === 'versionInfo' && (
-            // FIXME-A11Y
-            // eslint-disable-next-line jsx-a11y/anchor-is-valid
-            <a
-              href="#"
+            <button
               className="help-panel__sub-panel-link"
               onClick={e => openSubPanel(e, 'tutorial')}
+              aria-label="Show tutorial panel"
```

Comment:
Yes, I noticed the focus issue too, I think this may give more fuel for the eventual tab solution. I'm not sure how to force preact _not_ to rebuild the button if a prop changes on it, but if you have some tricks for that, please share.